spk/haproxy/src/app/application/db.py

```diff
@@ -74,7 +74,7 @@ def default_config():
     session.add(Backend(id=2, name=u'dsm', servers=u'dsm localhost:5000 check'))
     session.add(Backend(id=3, name=u'sabnzbd', servers=u'sabnzbd localhost:8080 check'))
     session.add(Backend(id=4, name=u'nzbget', servers=u'nzbget localhost:6789 check'))
-    session.add(Backend(id=5, name=u'sickbeard', servers=u'sickbeard localhost:8081 check'))
+    session.add(Backend(id=5, name=u'sickbeard', servers=u'sickbeard localhost:8083 check'))
```

Comment:
I do not agree to change port assigned to sickbeard. I recommend to assign sickchill to port 8083

Reply:
This is a fixed mistake. There is no SickBeard package, just sickbeard-custom which uses 8083